src/main/java/dk/cngroup/lentils/repository/StatusRepository.java

```diff
@@ -16,4 +17,8 @@
     List<Status> findByTeam(Team team);
     Boolean existsStatusByCypherAndTeam(Cypher cypher, Team team);
     void deleteAllByCypherCypherId(Long cypherId);
+    @Query(value = "SELECT DISTINCT s.* FROM status s " +
+            "WHERE s.cypher_status = 2 AND s.team_team_id = ?1",
```

Comment:
Jedna zveci souvisejici s Clean codem jsou tzv. magicke konstanty. Jsou to cisla, stringy, atd. ktere se nachazeji na ruznych mistech v kodu a je obtizne je v pripade potreby vsechny najit a menit. Proto se je snazime nepouzivat. Tady by bylo vhodnejsi:
```java
"WHERE s.cypher_status = " + CypherStatus.PENDING.ordinal() + " AND s.team_team_id = ?1"
```